source/includes/steps-install-shell-base.yaml

```diff
@@ -35,7 +35,7 @@ content: |
    .. code-block:: sh
 
       chmod +x bin/mongosh
-      chmod +x bin/mongocryptd-mongosh
+      chmod +x bin/mongosh_csfle_v1.so
```

Comment:
No need to do `chmod +x` for shared libraries :)

Reply:
Thanks for pointing this out Anna, I have removed the chmod commands for the shared libs.